src/templates/snippets/header.twig

```diff
@@ -38,7 +38,7 @@
     {{ _api.widgets.collection.get('notifications') }}
     <header class="Header" id="top">
         <div class="Container">
-            <div class="grid align-center justify-between">
+            <div class="grid nc-1 nc-2-xl align-center justify-between">
```

Comment:
@eharringtonaptuitiv This one is actually a 2-column layout on all sizes.

On small screens it's two rows with the first row having 2 columns and the second row having 1 column.
On x-large screen it is one row with two columns due to the `hidden-xl` class.